.drone/drone.jsonnet

```
@@ -153,7 +153,29 @@ local manifest(apps) = pipeline('manifest') {
     ],
   },
 ] + [
-  multiarch_image(arch)
+  multiarch_image(arch) + (
+    // When we're building Promtail for ARM, we want to use Dockerfile.arm32 to fix
+    // a problem with the published Drone image. See Dockerfile.arm32 for more
+    // information.
+    //
+    // This is really really hacky and a better more permanent solution will be to use
+    // buildkit.
+    if arch != 'arm' then {}
+    else {
+      steps: [
+        step + (
+          if step.name == 'build-promtail-image' || step.name == 'publish-promtail-image'
```

```suggestion
          if step.settings.dockerfile == "cmd/promtail/Dockerfile"
```

I think this one is safer, as it tests the actual condition, not the name, which might be modified

---

good idea, but i had to tweak it a little (not all steps had settings), see next commit